### ISSUE TITLE
aria-sort values table descriptions

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#1614]
since `aria-sort` can be specified on `columnheader` or `rowheader` it seems to me that the descriptions for 'ascending', 'descending' and 'none' should be updated to reference 'or row'.

For instance:
>Items are sorted in ascending order by this column **or row**.
